Derbil/ViewController.swift

```diff
@@ -117,8 +118,20 @@ class ViewController: UIViewController, WalkViewControllerDelegate, UIGestureRec
                 mouth: Face.Name.MouthName(Face.Mouth.Puppy, Face.Part.Mouth),
                 leftEye: Face.Name.EyeName(Face.Eye.Normal, Face.Part.Left),
                 rightEye: Face.Name.EyeName(Face.Eye.Normal, Face.Part.Right))
+            let center = CGPointMake(self.view.frame.midX, self.view.frame.midY)
+            
             self.faceView = FaceView(face: face, frame: self.faceContainer.bounds)
             self.faceContainer.addSubview(self.faceView!)
+            
+            let faceSnapBehavior = UISnapBehavior(item: self.faceView!, snapToPoint: center)
+            let headSnapBehavior = UISnapBehavior(item: self.headView, snapToPoint: CGPointMake(self.headView.frame.midX, self.headView.frame.midY))
+            
+            self.pushHeadBehavior = UIPushBehavior(items: [self.faceView!, self.headView], mode: .Continuous)
+            self.pushHeadBehavior!.pushDirection = CGVectorMake(0, 0)
+            self.pushHeadBehavior!.magnitude = 0
+            self.animator!.addBehavior(faceSnapBehavior)
+            self.animator!.addBehavior(headSnapBehavior)
+            self.animator!.addBehavior(self.pushHeadBehavior!)
```

Comment:
Adding these behaviors in viewDidAppear could cause bugs later if we have more than one view in the app.